Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -46,31 +83,23 @@ public override void OnPreSceneQuery()
 
             closestNormal = Rotation * Vector3.forward;
 
-            // Check proximity
-            BaseNearInteractionTouchable newClosestTouchable = null;
+            var layerMasks = PrioritizedLayerMasksOverride ?? PokeLayerMasks;
+
+            // Find closest touchable
+            ColliderNearInteractionTouchable newClosestTouchable = null;
+            foreach (var layerMask in layerMasks)
             {
-                closestDistance = float.PositiveInfinity;
-                foreach (var prox in BaseNearInteractionTouchable.Instances)
+                if (FindClosestTouchableForLayerMask(layerMask, out newClosestTouchable, out closestDistance, out closestNormal))
```

Comment:
Something to be careful of about touch interactions on HoloLens is people tend to move their hands quite far behind UI. For example, here is a capture of hand motion as somebody presses a button on a 2D slate:

![image](https://user-images.githubusercontent.com/168492/59759497-10935300-9245-11e9-9e55-8fad3c0b6943.png)

Notice how far behind the slate the person's finger moves.

If we go with this approach it will be important to keep touchableDistance quite high (~ .5,[ average arm length of male is 65 cm](https://www.reference.com/science/long-average-human-arm-62c7536c5e56f385)).

Reason is it would surprise people if they are for example scrolling and get a touch up event as they are scrolling the surface. The reason this could happen is people could move their hand quite far behind the slate because there's no physical surface stopping them.

I'm not sure a value of 0.2 will be high enough to cover most of the ways that people scroll content -- people will likely disengage by moving their hand too far back behind the surface.

Note that if the same variable for determining maximum touchable distance ALSO turns on the poke pointer visual and turns on input focus, then you will start getting focus events from poke pointers even when the pointer is very far away.

Reply:
This is the same default value as before, i.e. distFront on the BaseNearInteractionTouchable.